R/init.R

```diff
@@ -10,32 +10,29 @@
 #' End-users: This is the most important function for you to start a
 #' `teal` app that is composed of `teal` modules.
 #'
-#' @details
-#' When initializing the `teal` app, if `datanames` are not set for the `teal_data` object,
-#' defaults from the `teal_data` environment will be used.
```

Comment:
FYI:
moved the whole note into the `data` argument description. This applies only for it so no need to have a top-level note.